manifests/h/Hamdpac/gissaettnummer/1.0.0/Hamdpac.gissaettnummer.locale.svc.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Hamdpac.gissaettnummer
+PackageVersion: 1.0.0
+PackageLocale: svc
+Publisher: Hamdpac
+PackageName: gissa ett nummer
+License: copyrigt

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate